test/json_test.go

```diff
@@ -0,0 +1,41 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type MockT struct {
```

Comment:
Hmmm. Should this go into foxygo.at/s/mock. Half-serious, half-not.

Reply:
vetoed... then i'd have to implement all methods and I am way too lazy right now ;)